addons/rating/models/rating.py

```diff
@@ -41,7 +41,8 @@ def _default_access_token(self):
     parent_res_id = fields.Integer('Parent Document', index=True)
     rated_partner_id = fields.Many2one('res.partner', string="Rated person", help="Owner of the rated resource")
     partner_id = fields.Many2one('res.partner', string='Customer', help="Author of the rating")
-    rating = fields.Float(string="Rating Number", group_operator="avg", default=0, help="Rating value: 0=Unhappy, 10=Happy")
+    rating = fields.Float(string="Rating Number", group_operator="avg", default=0, help="Rating value: 0=Unhappy, 5=Happy")
+    rating_for_image = fields.Float(string="Adjusted Rating for Image", compute='_compute_rating_for_image', default=0, help="Rating floored to image value (1-3-5)")
```

Comment:
0 is already the default value for a Float field + we could initialize the column in a migration script.

Reply:
Done